int64 start
string generalImageFileName
string generalDepthImageName
---
bool success
